examples/tasks/.happy/terraform/envs/rdev/main.tf

```diff
@@ -0,0 +1,64 @@
+module "stack" {
```

Comment:
Are all of these `examples/tasks/.happy/terraform/envs/rdev/*` files generated by the happy infra generate command?

Reply:
No, you can tell because they have comments. Currently, `happy infra` is unable to write comments out.